gehomesdk/erd/values/hood/common_enums.py

```diff
@@ -0,0 +1,34 @@
+import enum
+from typing import NamedTuple, Optional
+
+@enum.unique
+class ErdHoodLightSetting(enum.Enum):
+    OFF = 0
+    DIM = 1
+    HIGH = 2
+
+    def boolify(self) -> Optional[bool]:
+        return self != ErdHoodLightStatus.OFF
+
+    def percentify(self) -> Optional[int]:
+        return int(self.name.title())*50
```

Comment:
Not sure where these will be used, but should it be self.value instead of self.name.title()?

Reply:
For percentify, we might need to add a method to ge_appliance.py similar to boolify_erd_value (perhaps percentify_erd_value?)  That would help us later when we're in the HA component.

Also, I think HA primarily uses a 0-1 range (float) for their percentages, perhaps we should convert percentify to return a float in this range instead of an int?

Reply:
See my comment above.

Reply:
OK, let's remove for now then and if you decide you need them later, we can re-add.

Reply:
> OK, let's remove for now then and if you decide you need them later, we can re-add.

Done!